.github/CONTRIBUTING.md

```diff
@@ -323,50 +323,6 @@ run our/your test with
 python -m pytest tests/..../...py::test_explain_what_is_being_tested --verbose --capture=no
 ```
 
-#### How to contribute bugfixes/features?
-
-Currently we have separate streams/branches for bugfixes/features and release from the default branch (`master`).
-Bugfixes should land in this `master` branch and features should land in `release/X.y-dev`.
-This means that when starting your contribution and creating a branch according to question 2) you should start this new branch from master or future release dev branch.
-Later in PR creation also pay attention to properly set the target branch, usually the starting (base) and target branch are the same.
-
-_Note, that this flow may change after the 1.2 release as we will adjust releasing strategy._
-
-#### How to fix PR with mixed base and target branches?
```

Comment:
would keep this section, as we still may have some PR from past branch which shall be now master

Reply:
@Borda But we could remove the `How to contribute bugfixes/features` section, right?

Reply:
yes, agree :]

Reply:
@Borda Done